docs/checkstyle/check-docs.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ue
+
+ret=0
+cd docs/guides
+
+for docs in admin developer; do
+  cd $docs
+  echo "Markdown doc $docs build log:"
+  mkdocs build &> mkdocs.log
```

Comment:
This just silently fails if `mkdocs` or any of the extensions is not installed. Not relevant for CI, but maybe confusing if devs try to execute it locally?